components/profile_picture.jsx

```diff
@@ -34,6 +37,12 @@ export default class ProfilePicture extends React.PureComponent {
     }
 
     render() {
+        const profileSrc = (typeof this.props.profileSrc === 'string' && this.props.profileSrc !== '') ?
```

Comment:
Can you elaborate on the need to distinguish `profileSrc` and `src`? I see they are both used below, but I can't quite figure out why we need two.

Reply:
`profileSrc` will be the original profile picture, and if not overriden, will equal to `src`. The need for this is for the popup of the profile to show the actual profile picture regardless of the post having an overriden icon. This way the 'identity' shown in the popup is of the actual user.

Reply:
Ah, subtle -- thank you! Perhaps if we used terms like `originalProfileSrc` and `profileSrc`? 0/5 on the right terms.

Reply:
I feel this to be a case where a comment in the code is deserved, so I added one. I think a variable name change would not be enough.